acceptance_tests/features/steps/edit_respondent_details.py

```diff
@@ -46,9 +36,9 @@ def change_contact_number(_):
     edit_respondent_details_form.edit_contact_number("01633 878787")
 
 
-@when('they change the email address')
-def edit_email_address(_):
-    edit_respondent_details_form.edit_email_address(Config.RESPONDENT_USERNAME)
+@when('they change the email address to "{email}')
```

Comment:
You are missing a close to the double quotes after the closing }. Other than that looks good.